src/api/timezones/timezones.ts

```diff
@@ -0,0 +1,35 @@
+import axios from 'axios';
+import { paths } from '../types/generated-schema';
+import { addError } from '../utils';
+
+const TIMEZONES_ENDPOINT = '/api/v1/i18n/timeZones';
+
+type ListTimeZoneNamesResponsePayload =
+  paths[typeof TIMEZONES_ENDPOINT]['get']['responses'][200]['content']['application/json'];
+
+export type TimeZone = ListTimeZoneNamesResponsePayload['timeZones'][0];
+
+type GetTimeZonesResponse = {
+  requestSucceeded: boolean;
+  timeZones: TimeZone[] | undefined;
+  error?: string;
+};
+
+export const getTimeZones = async (): Promise<GetTimeZonesResponse> => {
```

Comment:
oops, missed this.. it supports an optional locale
locale?: string
this is important